src/components/CategoryForm.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { Form, Loader, Button } from "@ahaui/react";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import categoryAPI from "api/categoryAPI";
+import { useTypedDispatch } from "hooks";
+import { NotiMsgType } from "store/actions/notiMsgActions";
+import { Category, CategoryPayload } from "types/category";
+import { closePopup } from "store/actions/popupActions";
+
+type CategoryFormProps = {
+  id?: number;
+  item?: CategoryPayload | null;
```

Comment:
```suggestion
  item?: CategoryPayload | null;
```
Can we only use null or undefined?